src/migrations/20201101132809-region.js

```diff
@@ -0,0 +1,61 @@
+module.exports = {
+  up: (queryInterface) => queryInterface.bulkInsert('Regions', [
+    {
+      id: 1,
+      name: '1',
```

Comment:
Do we need the name column if we are just using the number as the name? I could see keeping the name if the names were actual strings ("Boston"/"New York City"/etc...)

Reply:
Were you thinking about removing the `Regions `table?

Reply:
I think it is still useful to have a `regions` table. I'm not sure if there is a reason to have a `name` text column that just matches the value of the `id` column

Reply:
Yes, that's a low value. Except for the "No Region" name. I was thinking it would help the dropdown display if we had names. If you think the UI doesn't really care, we could remove the `name ` column

Reply:
Oh yeah I forgot about the "No Region" region. Maybe we should keep the name. Also I'm not sure how we'll handle that region on the frontend because it needs to be conditionally select-able, really should only be select-able in the admin UI when setting permissions.